### ISSUE TITLE
Add defensive code to HIVE_SERVER lookup in HiveGroovy

### ISSUE THREAD

**Issue by snowch** [IBM-Bluemix/BigInsights-on-Apache-Hadoop#27]
If the following lines fail, we could maybe add an error saying that HIVE_SERVER could not be found in Ambari lookup:

https://github.com/IBM-Bluemix/BigInsights-on-Apache-Hadoop/blob/master/examples/HiveGroovy/build.gradle#L61

```
environment 'hostname', getMasters(props)['HIVE_SERVER'][0]
```

**Comment by pregazzoni** [comment]
Fixed via commit c03a47e. It will raise exception if cannot find HIVE_SERVER.

**Comment by snowch** [comment]
Thanks @pregazzoni!
